lib/fluent/plugin/out_google_cloud.rb

```diff
@@ -70,6 +70,12 @@ class GoogleCloudOutput < BufferedOutput
     # component (Docker, Kubelet, etc.) logs from container logs.
     config_param :detect_subservice, :bool, :default => true
 
+    # The regular expression to use on Kubernetes logs to extract some basic
+    # information about the log source. The regex must contain capture groups
+    # for pod_name, namespace_name, and container_name.
+    config_param :kubernetes_tag_regexp, :string, :default =>
+      '\.(?<pod_name>[^_]+)_(?<namespace_name>[^_]+)_(?<container_name>.+)$'
```

Comment:
Why not include `var.log.containers.` prefix in there as default? It's the default for kubernetes file path AFAIK so should be a good default IMO.